marker_scale: 1.0
marker_thickness: 1.0
home_coordinates: [10.0, 78.0]
x_coordinates: [25.0, 35.0, 45.0, 45.0, 35.0, 25.0, 17.5]
y_coordinates: [75.0, 75.0, 75.0, 70.0, 70.0, 70.0, 63.0]
